docs/package.json

```diff
@@ -22,7 +22,6 @@
     "@storybook/addon-essentials": "^6.5.12",
     "@storybook/addon-interactions": "^6.5.12",
     "@storybook/addon-links": "^6.5.12",
-    "@storybook/addon-storyshots": "^6.5.15",
     "@storybook/addon-storysource": "^6.5.12",
     "@storybook/builder-vite": "^0.2.2",
```

Comment:
suggest updating this to latest (`0.3.0`) to benefit from this in the storybook build - I don't know why it requires a version upgrade, but upgrading it resolved the issue on https://github.com/AdaptiveConsulting/early-careers-ui-kit-1